cypress/e2e/tests/migration/task-manager/cancel_task_negative.test.ts

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
```

Comment:
If More negative tests can be added to this file later , let's rename it to negative_test.ts